src/java.base/share/classes/sun/nio/ch/DirectBuffer.java

```diff
@@ -30,6 +30,17 @@
 
 public interface DirectBuffer {
 
+    // Use of the returned address must be guarded if this DirectBuffer
+    // is backed by a memory session that is explicitly closeable.
+    //
+    // Failure to do this means the outcome is undefined including
+    // silent unrelated memory mutation and JVM crashes.
+    //
+    // An example of a guarded use of a memory address is shown here:
+    //
+    // try (var guard = NIO_ACCESS.acquireSession(bb)) {
+    //     performOperation(bb.address());
+    // }
```

Comment:
Again, updating this comment was missed:
```suggestion
    // try (var guard = NIO_ACCESS.acquireScope(bb)) {
    //     performOperation(guard.address());
    // }
```